tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -440,76 +441,82 @@ def test_fast_capacity_failover(
     # "ice-compute-resource" and "ice-cr-multiple" compute resources
     remote_command_executor.run_remote_script(str(test_datadir / "overrides.sh"), run_as_root=True)
 
-    # All nodes
-    nodes_in_scheduler = scheduler_commands.get_compute_nodes("queue1", all_nodes=True)
-    static_nodes, dynamic_nodes = get_partition_nodes(nodes_in_scheduler)
+    for strategy in scaling_strategies:
+        partition = f"queue-{strategy}"
+        # All nodes
+        nodes_in_scheduler = scheduler_commands.get_compute_nodes(partition, all_nodes=True)
+        static_nodes, dynamic_nodes = get_partition_nodes(nodes_in_scheduler)
```

Comment:
It doesn't seems the strategy is involved here.

Are we changing the cluster ?
If not we can retrieve this set only once outside the for loop as done before.